tools/GenerateHeaderForJson.ps1

```diff
@@ -0,0 +1,28 @@
+# This script is used for taking a json file and stamping it into a header with
+# the contents of that json files as a constexpr string_view in the header.
+
+param (
+    [parameter(Mandatory=$true, Position=0)]
+    [string]$JsonFile,
+
+    [parameter(Mandatory=$true, Position=1)]
+    [string]$OutPath,
+
+    [parameter(Mandatory=$true, Position=2)]
+    [string]$VariableName
+)
+
+# Load the xml files.
+$jsonData = Get-Content $JsonFile
+
+Write-Output "// Copyright (c) Microsoft Corporation" | Out-File -FilePath $OutPath

```

Comment:
i think you'll end up writing a UTF-16 file. Do you want that? You'll need to add -Encoding ASCII or something to every out-file